addons/dialogue_manager/components/editor_property/editor_property_control.tscn

```diff
@@ -1,9 +1,21 @@
-[gd_scene load_steps=4 format=3 uid="uid://ycn6uaj7dsrh"]
+[gd_scene load_steps=6 format=3 uid="uid://ycn6uaj7dsrh"]
 
 [ext_resource type="Script" uid="uid://dooe2pflnqtve" path="res://addons/dialogue_manager/components/editor_property/editor_property_control.gd" id="1_het12"]
 [ext_resource type="PackedScene" uid="uid://b16uuqjuof3n5" path="res://addons/dialogue_manager/components/editor_property/resource_button.tscn" id="2_hh3d4"]
 [ext_resource type="PackedScene" uid="uid://dnufpcdrreva3" path="res://addons/dialogue_manager/components/files_list.tscn" id="3_l8fp6"]
 
+[sub_resource type="Image" id="Image_vs2uv"]
```

Comment:
Please do not modify the `dialogue_manager` addon. For one thing, this change will be overwritten when the addon is updated.